test/e2e/env.go

```diff
@@ -34,7 +34,7 @@ func (e Env) Validate(t *testing.T) {
 	errStrs := []string{}
 
 	if len(e.Namespace) == 0 {
-		errStrs = append(errStrs, "Expected Namespace to be non-empty")
+		errStrs = append(errStrs, "Expected Namespace to be non-empty (Hint: set ENV var KAPP_E2E_NAMESPACE")
```

Comment:
Should we specify that the value should be set to `kapp-test`?
This value is hard-coded in a bunch of test cases if not most of them.

Reply:
sgtm;   I added an `=kapp-test`